src/K9OCRS/ClientApp/src/areas/applications/Confirmation/index.js

```diff
@@ -1,74 +1,98 @@
-import React from "react";
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+import { connect } from "react-redux";
+import { useParams } from "react-router-dom";
+import selectors from "../../../shared/modules/selectors";
 
-const Confirm = () => {
+
+const Confirm = props => {
+
+  const { sectionId } = useParams();
+  const [sectionDetail, setSectionDetail] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [alerts, setAlerts] = useState([]);
```

Comment:
The `loading` and `alerts` variables are currently unused so they cause linting warnings. I know they'll probably be used when you revisit this page later on so we can just hide the linting warnings for now. Remember to keep an eye on the npm console to avoid linting warnings or errors.
![image](https://user-images.githubusercontent.com/30603929/156304311-3ab80913-60d9-42fc-8652-bc79dad81998.png)


```suggestion
  // TODO: remove the eslint comments from the lines below and actually use the loading and alerts variables.
  const [loading, setLoading] = useState(true); // eslint-disable-line
  const [alerts, setAlerts] = useState([]); // eslint-disable-line
```